propagators/b3/hs-opentelemetry-propagator-b3.cabal

```diff
@@ -24,12 +26,23 @@ source-repository head
   location: https://github.com/iand675/hs-opentelemetry
 
 library
+  exposed-modules:
+      OpenTelemetry.Propagator.B3
+      OpenTelemetry.Propagator.B3.Internal
   other-modules:
       Paths_hs_opentelemetry_propagator_b3
   hs-source-dirs:
       src
+  ghc-options: -Wall
   build-depends:
-      base >=4.7 && <5
+      attoparsec
+    , base >=4.7 && <5
+    , bytestring
+    , hs-opentelemetry-api ==0.0.3.*
```

Comment:
Please update this _cabal_ file with the current _package.yaml_. `case-insensitive` is not included.

Reply:
Oh woops, we don't actually need `case-insensitive` and I removed it from the cabal file directly rather then `package.yaml`. I don't usually use hpack.